tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -328,23 +328,63 @@ describe(__filename, () => {
     expect(root.find('.InstallButtonWrapper-download')).toHaveLength(1);
   });
 
-  it('does not display a download link when the browser is compatible', () => {
+  it('does not display a download link when the browser is compatible and showLinkInsteadOfButton is false', () => {
     const _findInstallURL = sinon
       .stub()
       .returns('https://a.m.o/files/addon.xpi');
     const _getClientCompatibility = sinon.stub().returns({
       compatible: true,
     });
+    const showLinkInsteadOfButton = false;
```

Comment:
You don't really need a `const` for this. You could just include `showLinkInsteadOfButton: false` on line 344 below.